notifications_utils/jinja_templates/letter_image_template.jinja2

```diff
@@ -1,3 +1,14 @@
+{% if too_many_pages %}
+  <div class='banner-dangerous bottom-gutter'>
+    <h1 class='banner-title'>
+      This letter is too long
+    </h1>
+    <p>
+      Letters can’t have more than {{ page_numbers[-1] }} pages
```

Comment:
GOV.UK style guide advises against negative contractions: https://www.gov.uk/guidance/content-design/writing-for-gov-uk#contractions
I think it's a bit more nuanced than that – context is important – but we probably should avoid them in error messages, where the instruction must be explicit.

Suggestion:
`Letters cannot be more than {{ page_numbers [-1] }} pages long`

Reply:
![image](https://user-images.githubusercontent.com/355079/44407890-1b168a80-a557-11e8-855e-fd023395db3c.png)